JapaneseFilter/sections/specific.txt

```diff
@@ -4663,3 +4663,5 @@ gtoys.blog.fc2.com##.adsbygoogle
 gtoys.blog.fc2.com###adleft
 !+ PLATFORM(ext_ff, ext_opera, ios, ext_android_cb, ext_ublock)
 gtoys.blog.fc2.com###adright
+! https://github.com/AdguardTeam/AdguardFilters/issues/133556
+oshiete.goo.ne.jp###connection_area > div.clearfix
```

Comment:
Don't add the rule with the comment at the end of the file - do it at the beginning after the common rule `! ##.ad-banner`.

Reply:
do i need to sort from a-z?

Reply:
No, it's not needed.